src/runtime/components/Table.vue

```diff
@@ -231,7 +232,7 @@ defineExpose({
         <tr v-else :class="ui.tr({ class: [props.ui?.tr] })">
           <td :colspan="columns?.length" :class="ui.empty({ class: props.ui?.empty })">
             <slot name="empty">
-              No results
+              {{ t('ui.table.noData') }}
```

Comment:
I'd also keep `No results` naming here, this might happen due to a search.

Reply:
Yes, because of the lookup it could be, but then this problem should be solved by the user passing the `#empty' slot.  Exactly the same way we solve it in other components. 

I don't think it's necessary to replace it with `No results`, since in the vast majority of cases it's `No data` that's needed.